default/scripting/species/common/general.macros

```diff
@@ -13,15 +24,19 @@ FOCUS_CHANGE_PENALTY
             activation = And [
                 Planet
                 Not Focus type = "FOCUS_INDUSTRY"
+                (0 <= Source.TurnsSinceFocusChange < 1)
```

Comment:
Significance of writing it this way?
Seems to activate only on a value of 0, so the effect would always evaluate to `Value - 1`

Reply:
The original idea of the focus change penalty was that it would start out large and then decrease in magnitude over several turns. For now just `(0 == Source.TurnsSinceFocusChange)` is probably OK.